api-server/src/main/resources/swagger.json

```diff
@@ -1,2861 +1,2872 @@
+        },
+        "io.enmasse.admin.v1beta2.AddressSpacePlan": {
+            "properties": {
+                "apiVersion": {
+                    "enum": [
+                        "admin.enmasse.io/v1beta2"
+                    ],
+                    "type": "string"
+                },
+                "kind": {
+                    "enum": [
+                        "AddressSpacePlan"
+                    ],
+                    "type": "string"
+                },
+                "metadata": {
+                    "$ref": "#/definitions/ObjectMeta"
+                },
+                "spec": {
+                    "$ref": "#/definitions/io.enmasse.admin.v1beta2.AddressSpacePlanSpec"
+                }
+            },
+            "required": [
+                "apiVersion",
+                "kind",
+                "metadata",
+                "spec"
+            ],
+            "type": "object"
+        },
+        "io.enmasse.admin.v1beta2.AddressSpacePlanList": {
+            "properties": {
+                "apiVersion": {
+                    "enum": [
+                        "admin.enmasse.io/v1beta2"
+                    ],
+                    "type": "string"
+                },
+                "items": {
+                    "items": {
+                        "$ref": "#/definitions/io.enmasse.admin.v1beta2.AddressSpacePlan"
                     },
-                    "404": {
-                        "description": "Not found"
-                    }
+                    "type": "array"
+                },
+                "kind": {
+                    "enum": [
+                        "AddressSpacePlanList"
+                    ],
+                    "type": "string"
                 }
             },
-            "put": {
-                "tags": [
-                    "enmasse_v1beta2",
-                    "admin",
-                    "addressspaceplan"
-                ],
-                "description": "replace the specified AddressSpacePlan",
-                "operationId": "replaceAdminEnmasseV1beta2NamespacedAddressSpacePlan",
-                "produces": [
-                    "application/json"
-                ],
-                "parameters": [
-                    {
-                        "name": "name",
-                        "in": "path",
-                        "description": "Name of AddressSpacePlan to replace.",
-                        "required": true,
+            "required": [
+                "apiVersion",
+                "kind",
+                "items"
+            ],
+            "type": "object"
+        },
+        "io.enmasse.admin.v1beta2.AddressSpacePlanSpec": {
+            "properties": {
+                "addressPlans": {
+                    "items": {
                         "type": "string"
                     },
-                    {
-                        "in": "body",
-                        "name": "body",
-                        "required": true,
-       
```

Comment:
It is nice to see the api verison near the top.

Reply:
Interesting.  I added the comment to the addition of the swagger: 2.0 line.  And it showed the matched removed lines.

Reply:
Fixed the ordering on top-level fields now. I think the order within the definitions and paths isn't that important.